Source/WebCore/css/CSSPageRule.cpp

```diff
@@ -56,11 +56,10 @@ CSSStyleDeclaration& CSSPageRule::style()
 String CSSPageRule::selectorText() const
 {
     if (auto* selector = m_pageRule->selector()) {
-        String pageSpecification = selector->selectorText();
-        if (!pageSpecification.isEmpty() && pageSpecification != starAtom())
-            return makeString("@page "_s, pageSpecification);
+        if (!selector->selectorText().isEmpty())
+            return selector->selectorText();
     }
-    return "@page"_s;
+    return ""_s;
```

Comment:
This should probably use `emptyString()`?

Reply:
At one time that was slightly more efficient. Not sure if that's still the case.